Sources/Rendering/Core/ImageMapper/index.js

```diff
@@ -427,17 +425,34 @@ function vtkImageMapper(publicAPI, model) {
     }
     return null;
   };
+
+  publicAPI.setSliceAtFocalPoint = (flagValue, camera) => {
+    model.sliceAtFocalPoint = flagValue;
+
+    // Unsubscribe from previous callback, if any.
+    if (model.cameraSubscription) {
+      model.cameraSubscription.unsubscribe();
+      model.cameraSubscription = null;
+    }
+
+    // Setup new subscription to listen to camera modifications.
+    if (flagValue && camera) {
+      model.cameraSubscription = camera.onModified(() =>
+        publicAPI.setSliceFromCamera(camera)
+      );
+    }
+  };
```

Comment:
@floryst Does this change look good? Basically I am directly hooking up the `setSliceFromCamera()` function with the `onModified` event of the Camera. This is to compensate for removing the call from the OpenGLImageMapper.

Reply:
@jourdain @finetjul what do you think about pulling out the setSliceAtFocalPoint logic from the OpenGLImageMapper? I think we are going to keep the original functionality, but it does mean that the OpenGLImageMapper must handle special cases when dealing with either ImageMapper or ImageArrayMapper.

Reply:
Generally speaking, I do not like when a mapper modifies the camera. It is prone to hard-to-debug bugs.

Reply:
In this case, the mapper is dependent on the camera for information. We have [the OpenGL mapper setting properties on the Core mapper](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/OpenGL/ImageMapper/index.js#L67-L70), and Raj was looking into doing this update via a camera onModified listener.

Reply:
I guess, I will just revert to previous behavior. It's not directly related to this PR, so we can address this later.